html/Mogo/js/fetchAPI/helper-functions.js

```diff
@@ -0,0 +1,30 @@
+function getMonthByValue(value) {
+  const months = {
```

Comment:
it is possible to have just array and get value by index, isn't this structure looks like array?